scheduler/test/cook/test/pool.clj

```diff
@@ -1,7 +1,8 @@
 (ns cook.test.pool
   (:require [clojure.test :refer :all]
             [cook.config :as config]
-            [cook.pool :as pool])
+            [cook.pool :as pool]
+            [clojure.tools.logging :as log])
```

Comment:
Lexicographic

Reply:
Actually do you need this new `:require`?

Reply:
nope, deleted